### PR TITLE
Add recipe for zenpy

### PR DESCRIPTION
**Pull request by xylar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xylar** [comment]
This is needed for https://github.com/conda-forge/apache-airflow-providers-zendesk-feedstock/pull/3

**Comment by xylar** [comment]
@conda-forge/help-python, this is ready to review.  Please let me know if you have any concerns.  Most of this recipe is straight from grayskull so the review should hopefully be easy.

File: recipes/zenpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - zenpy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/facetoe/zenpy
+  summary: Python wrapper for the Zendesk API
+  license: GPL-3.0-only
```
Comment (BastianZim) [inline recipes/zenpy/meta.yaml:40]: FYI: https://github.com/facetoe/zenpy/pull/504

This should include the license file in the next release.

**Comment by xylar** [comment]
@BastianZim, thank you very much for merging and for your tip. I would likely not have noticed.

**Pull request merged**